ultraScurve.py

```diff
@@ -140,51 +98,52 @@
             trimVal = (0x3f & readVFAT(ohboard,options.gtx,vfat,"VFATChannels.ChanReg%d"%(scCH)))
             writeVFAT(ohboard,options.gtx,vfat,"VFATChannels.ChanReg%d"%(scCH),trimVal)
 
+    gemData.mode[0] = scanmode.SCURVE
     for scCH in range(CHAN_MIN,CHAN_MAX):
-        vfatCH[0] = scCH
+        gemData.vfatCH[0] = scCH
         print "Channel #"+str(scCH)
         for vfat in range(0,24):
             if (mask >> vfat) & 0x1: continue
             trimVal = (0x3f & readVFAT(ohboard,options.gtx,vfat,"VFATChannels.ChanReg%d"%(scCH)))
             writeVFAT(ohboard,options.gtx,vfat,"VFATChannels.ChanReg%d"%(scCH),trimVal+64)
-        configureScanModule(ohboard, options.gtx, scanmode.SCURVE, mask, channel = scCH,
+        configureScanModule(ohboard, options.gtx, gemData.mode[0], mask, channel = scCH,
```

Comment:
Might consider (with the functions mentioned in the summary) leaving this as `scanmode.SCURVE` and changing the `gemData` constructor to take the `scanmode.<TYPE>` as a parameter (instead of line 101)
Leaves this configure function call clear, but still puts the desired information in the output tree

Reply:
Done